website/pages/input-unions.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Input Unions
```

Comment:
`@oneOf` is not equivalent to "input union". You can _use_ it to form an input union, but its use is broader than that. I would title it "OneOf input objects" or similar.

(Example: `input UserSpecifier @oneOf { databaseId: Int, username: String, hospitalNumber: Int }` - note that both `databaseId` and `hospitalNumber` are `Int` and can uniquely identify a user in different ways - so this isn't simply a union.)

Reply:
My hesitancy was mainly that it's less of a clickthrough as the name is opaque to people while a general known word helps. I do agree that they are not the same

Reply:
I agree; perhaps the URL would be oneof and the title would be "OneOf and input unions" or similar?